app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -123,19 +123,29 @@ class EcsCapacityProvider extends React.Component<IEcsCapacityProviderProps, IEc
     const capacityProviderStrategy = this.state.capacityProviderStrategy;
     const useDefaultCapacityProviders = this.state.useDefaultCapacityProviders;
     const capacityProviderLoadedFlag = this.state.capacityProviderLoadedFlag;
-    const dirtyCapacityProviders = this.props.command.viewState.dirty && this.props.command.viewState.dirty.capacityProviders ? this.props.command.viewState.dirty.capacityProviders : [];
+    const customDirtyCapacityProviders = this.props.command.viewState.dirty && this.props.command.viewState.dirty.customCapacityProviders ? this.props.command.viewState.dirty.customCapacityProviders : [];
+    const defaultDirtyCapacityProviders = this.props.command.viewState.dirty && this.props.command.viewState.dirty.defaulCapacityProviders ? this.props.command.viewState.dirty.defaulCapacityProviders : [];
 
     const capacityProviderNames = this.state.availableCapacityProviders &&  this.state.availableCapacityProviders.length > 0 ?  this.state.availableCapacityProviders.map((capacityProviderNames) => {
       return { label: `${capacityProviderNames}`, value: capacityProviderNames };
     }) : [];
 
-    const dirtyCapacityProviderList = dirtyCapacityProviders ? dirtyCapacityProviders.map(function (capacityProvider, index){
+    const defaultCPError = useDefaultCapacityProviders && defaultDirtyCapacityProviders ? (
+      <div className="alert alert-warning">
+        <p className="text-left">
+          <i className="fa fa-exclamation-triangle"></i>
+          Invalid capacity providers are a part of default capacity provider strategy.
+          <br/> Please click Done button to remove dirty capacity providers from the pipeline JSON.
```

Comment:
I'd like to make this text explain the end state vs. what's going to be removed. e.g.,:
"Please click 'Done' to use current default capacity provider strategy (shown below) or switch to using a custom strategy"